Rakefile

```diff
@@ -44,6 +44,7 @@ end
 desc 'Mutation testing to check mutation coverage of current RSpec test suite'
 task :mutant do
   mutant_sh = 'bundle exec mutant \
+    --since 18069a6bb8779c2381b361738c866ce4c35f7466 \
```

Comment:
mutant's `--since` supports any `git-reference`. I'd prefer to see a recognisable tag here, e.g. the latest release or similar.